CHANGES.rst

```diff
@@ -74,10 +74,8 @@ New Features
     browsing capabilities. To write out to this format, use
     ``Table.write('filename.html', format='jsviewer')``. [#2875]
 
-  - A ``quantity`` property and ``to`` method were added to ``Table``
```

Comment:
I'm not sure why this comment is being removed here, perhaps it is a funny that is related to the required rebase.